src/pytest_cov/plugin.py

```diff
@@ -47,6 +47,54 @@ def __call__(self, parser, namespace, values, option_string=None):
         namespace.cov_report[report_type] = file
 
 
+class FailUnder(object):
```

Comment:
I don't understand what this class is for, or does.  Docstrings?

Reply:
This is to abstract the limit, include and exclude for each `--cov-fail-under`

Reply:
It also doesn't need a docstring because it's internal

Reply:
Docstrings are helpful for people reading the code to understand what it does, whether it's internal or externally exposed.